Userland/Applications/FileManager/main.cpp

```diff
@@ -1164,6 +1176,7 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
     TRY(directory_context_menu->try_add_action(cut_action));
     TRY(directory_context_menu->try_add_action(copy_action));
     TRY(directory_context_menu->try_add_action(folder_specific_paste_action));
+    TRY(directory_context_menu->try_add_action(copy_path_action));
```

Comment:
I feel that the `Copy Path` action should immediately follow the `Copy` action in the menu. Same for the tree view context menu and file context menu.

Reply:
Done!